src/shared/components/Modal/Modal.jsx

```diff
@@ -107,6 +109,11 @@ const Modal = ({
               {secondaryText}
             </SecondaryButton>
           )}
+          {thirdText && (
```

Comment:
It doesn't look good
<img width="348" alt="image" src="https://user-images.githubusercontent.com/29302746/183301088-5f9c9ba5-3bc7-4d1b-b61a-e4fa81199ae7.png">
try to make it look like this:
<img width="383" alt="image" src="https://user-images.githubusercontent.com/29302746/183301191-fa91e4d3-1b40-4f03-813b-e17a8a419b06.png">

use this grid in the `Actions` component:
```
    display: grid;
    grid-template-columns: 1fr 1fr;
    grid-gap: 8px;
```
+ more changes in the buttons style